storage/ndb/src/kernel/ndbd.cpp

```diff
@@ -595,11 +595,11 @@ get_multithreaded_config(EmulatorData& ed)
     return 0;
 
   g_eventLogger->info("NDBMT: ldm_workers=%u ldm_threads=%u"
-                      " query_workers=%u query_threads=%u\n"
+                      " query_workers=%u\n"
                       " tc_threads=%u tc_workers=%u"
                       " send=%u receive=%u main_threads=%u",
                       globalData.ndbMtLqhWorkers, globalData.ndbMtLqhThreads,
-                      globalData.ndbMtQueryWorkers, globalData.ndbMtQueryThreads,
+                      globalData.ndbMtQueryWorkers,
                       globalData.ndbMtTcWorkers, globalData.ndbMtTcThreads,
```

Comment:
Do these two variables' positions need to be exchanged?

Reply:
Not sure why that should be required, can u expand on your reasoning?

Reply:
the format is defined as "tc_threads = %u tc_workers = %"
so I think we should keep that order

Reply:
ok, can do that, saw the issue now :)